Update Maskar's Oblivion Overhaul

Sabre142 opened the pull request.

Sabre142 committed:
Update Maskar's Oblivion Overhaul

* Move mod entry
 - To the Gameplay - General section.

* Update location
 - Add trailing slash.
 - Add mod name.

* Add requirement
 - Missing SI master. MOOEventTreeAncient01
   has SEGnarlBark item and SELpGnarlSilence5 spell.

* Add # doNotClean comment
 - Version 4.9.4.2.
 - Maskar advises not to clean the 1 ITM that was generated by the CS:
   https://forums.nexusmods.com/index.php?/topic/787622-maskars-oblivion-overhaul/?p=97912403

MacSplody commented on masterlist.yaml:
```diff
@@ -6690,6 +6690,16 @@ plugins:
+      - link: 'https://www.nexusmods.com/oblivion/mods/42780/'
+        name: 'Maskar''s Oblivion Overhaul'
+    group: *lateChangesGroup
+    after: [ 'Bashed Patch, 0.esp' ]  # The default filename for a Bashed Patch.
```
This load after rule could be removed, the group already puts it after the bashed patch.
```
    after: [ 'Bashed Patch, 0.esp' ]  # The default filename for a Bashed Patch.
```

Sabre142 committed:
Update Av Latta Magicka - Oblivion Magic Overhaul

* Remove load after rule
 - This load after rule can be removed, the
   group already puts it after the bashed patch.

MacSplody commented:
Thanks.
